.ci/update-github-page.sh

```diff
@@ -73,7 +73,7 @@ CONTENT=$(cat github_post.txt)
 echo "$CONTENT"
 
 echo 'Updating content to be be json value friendly'
-UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt |  sed "s/\`/'/g")
+UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt |  sed "s/[\`\"]/'/g")
```

Comment:
This step in general very strange, as it change actual content of post. 
can we escape only `"`?
and forbid '`' in release notes generator or do map of this symbol in releaasenotes builder ?

Reply:
We can forbid '`' in release notes generator. Let me know if I shall proceed with a PR for that. I personally would leave this step just in case.

Reply:
we can try to make file for json, without evaluation of it by `$`
https://stackoverflow.com/a/7173621

`-D @path-to-json`

So, lets concatenate file by 3 blocks
```
{
"tag_name": "checkstyle-${TARGET_VERSION}",
"body": "
```
`${UPDATED_CONTENT}`
```
"
}
```

we still need escape `"`.
But it will be not a changing it to `'`.

Reply:
We need to update relasenotes generator to fail with error if such symbol exists or escape it as we did with `@`

Reply:
fyi:
```
$ echo -n '" symbol' | jq -Rsa .
"\" symbol"
```

Reply:
Done https://github.com/stoyanK7/checkstyle/actions/runs/3525663711

> We need to update relasenotes generator to fail with error if such symbol exists or escape it as we did with @.

On it.